src/core/annotation.js

```diff
@@ -489,7 +492,22 @@ class Annotation {
    *                        4 (CMYK) elements
    */
   setColor(color) {
-    this.color = getRgbColor(color);
+    this.color = getRgbColor(color, new Uint8ClampedArray(3));
+  }
+
+  /**
+   * Set the color for background and border if any.
+   * The default values are transparent.
+   *
+   * @public
+   * @memberof Annotation
+   * @param {Dict}  - The MK dictionary
+   */
+  setBorderAndBackgroundColors(mk) {
+    if (mk instanceof Dict) {
+      this.borderColor = getRgbColor(mk.getArray("BC"), undefined);
```

Comment:
Transparent is indicated by `null` instead of `undefined` as indicated on line 202.

Reply:
I didn't find in the spec what's the default color (transparent or whatever), so `undefined` stands for "use HTML default".

Reply:
Wait, does that mean that we now use both `undefined` *and* `null`, but they have very slightly different meaning in the display-layer?
If so, that sounds like a recipe for confusion...